### PR TITLE
Support only props in InertiaLink

### PR DESCRIPTION
**Pull request by natsathorn**
@reinink @sebastiandedeyne 
This PR created alongside of this [pull request](https://github.com/inertiajs/inertia-react/pull/57).

### CODE REVIEW

**Comment by sebastiandedeyne** [comment]
Thanks!

**Comment by roarkmccolgan** [comment]
Hey Guys,
I'm getting the following warning:
`[Vue warn]: Invalid default value for prop "only": Props with type Object/Array must use a factory function to return the default value.`

@natsathorn I [believe](https://github.com/vuejs/vue/issues/1032#issuecomment-120212888) we need to declare default empty arrays / objects like this:

```javascript
props: {
  only: {
    type: Array,
    default: function () { return [] }
  }
}
```
or ES6

```javascript
props: {
  only: {
    type: Array,
    default: () => []
  }
}
```

Thanks

File: src/link.js [hunk 1]
```diff
@@ -27,6 +27,10 @@ export default {
       type: Boolean,
       default: false,
     },
+    only: {
+      type: Array,
+      default: [],
```
Comment (roarkmccolgan) [inline src/link.js:32]:
```
default: function () { return [] }
```

**Comment by Juhlinus** [comment]
@roarkmccolgan This issue is being discussed in #132 
I've got a PR ready here #133

**Comment by roarkmccolgan** [comment]
Thanks @Juhlinus

**Comment by sebastiandedeyne** [comment]
Released a fix in `0.1.4 ` :)

**Comment by natsathorn** [comment]
I will be more careful next time. Thank for letting me know.

**Pull request merged**